### PR TITLE
add docs on authenticating to prometheus via oauth proxy

### PR DESCRIPTION
**Pull request by jeremyeder**
Add some brief notes on how to authenticate to the oauth proxy.

### CODE REVIEW

File: examples/prometheus/README.md [hunk 1]
```diff
@@ -5,9 +5,28 @@ This template creates a Prometheus instance preconfigured to gather OpenShift an
...
+$ oc new-app -f https://raw.githubusercontent.com/openshift/origin/master/examples/prometheus/prometheus.yaml
 ```
 
+This template sets up an oauth proxy for authentication.  The oauth-proxy relies on the same authentication system configured for your OpenShift cluster.
+
+See [this documentation](https://docs.openshift.org/latest/install_config/configuring_authentication.html) for authentication options for OpenShift.
+
+One option documented above is htpasswd.  Create an htpasswd user called (for example) ```prometheus```.
+
+The ```prometheus``` user must have cluster-reader access.  Grant the user cluster-reader access:
```
Comment (smarterclayton) [inline examples/prometheus/README.md]: Actually only needs view access on the namespace that has the route.  Would recommend `oc policy add-role-to-user view -n kube-system prometheus`

**Comment by enj** [comment]
@openshift/sig-security

File: examples/prometheus/README.md (hunk 1 above)
Comment (jeremyeder) [reply examples/prometheus/README.md]: OK, I thought I had seen it here:  https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/tasks/install_prometheus.yaml#L48

I'll test out your suggestion to double check that things still work.

Comment (jeremyeder) [reply examples/prometheus/README.md]: That does work.

**Comment by enj** [comment]
@jeremyeder https://github.com/openshift/openshift-ansible/pull/5564 is probably of interest to you.

File: examples/prometheus/README.md (hunk 1 above)
Comment (zmhassan) [reply examples/prometheus/README.md]: Would be nice to include this in the readme for openshift-prometheus. I'll send a PR.

**Pull request closed without merging**